README.md

```diff
@@ -51,10 +51,10 @@ for cookie in cookies:
 ## Contributors
 
 * [ Angad ](https://github.com/L04DB4L4NC3R)
+* [ Ubaid ](https://github.com/Geek-ubaid/)
```

Comment:
Only add names of the developers, not maintainers

Reply:
ok but i got help from u 2